src/FSharpVSPowerTools.Core/OpenDeclarationsGetter.fs

```diff
@@ -348,7 +348,7 @@ module OpenDeclarationGetter =
             let res = 
                 match symbolUse.SymbolUse.Symbol with
                 | Pattern | RecordField _
-                | Entity (Class | (AbbreviatedType _ | FSharpType | ValueType | FSharpModule | Array), _, _)
+                | Entity (Class | Intf | (AbbreviatedType _ | FSharpType | ValueType | FSharpModule | Array), _, _)
```

Comment:
Why not use `Entity(_, Interface, _)` similar to `Entity(_, _, Tuple)` below?

The `Intf` active patterns seem very superficial.